backend/prisma/schema.prisma

```diff
@@ -20,15 +20,25 @@ model Game {
     playerMax       Int
     playtimeMinutes Int // Stored in minutes @map("playtime")
     location        String
-
-    borrow      Borrow[]
-    request     BorrowRequest[]
-    GameOwner   GameOwner?      @relation(fields: [gameOwnerId], references: [id])
-    gameOwnerId String?
+    playStatus      PlayStatus[]
+    borrow          Borrow[]
+    request         BorrowRequest[]
+    GameOwner       GameOwner?      @relation(fields: [gameOwnerId], references: [id])
+    gameOwnerId     String?
 
     @@map("games")
 }
 
+model PlayStatus {
+    id     String @id @default(cuid())
```

Comment:
Should we be using an ID on something that can already be identified with just a `User` and `Game`? I don't see those values having reason to change down the line, so unless we're planning to remove an entry entirely to mark a player as no longer having played it, I dont see its use.

Reply:
Yes